codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -50,15 +50,15 @@ public String willTopic() {
         return willTopic;
     }
 
-    public String willMessage() {
+    public byte[] willMessage() {
```

Comment:
Please use the same approach (add a new method) as in https://github.com/netty/netty/pull/6769 and avoid breaking the public API.